ExpenseTrackerGrupo2.csproj

```diff
@@ -8,9 +8,17 @@
 
   <ItemGroup>
     <PackageReference Include="Dapper" Version="2.1.35" />
+    <PackageReference Include="DotNetEnv" Version="3.1.1" />
     <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="8.0.6" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="9.0.0-rc.1.24431.7" />
```

Comment:
Mmm.... there is a lot of new tools  on csproj, maybe one or two it can be understandable, but there its actually more than 5, why its needed all the new tools?

Reply:
I was trying another solution, for what you already did and I forgot to delete it, but I already deleted it.